buildSrc/src/test/kotlin/com/datadog/gradle/plugin/jsonschema/ModelWithAdditionalAttributesTest.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.jsonschema
+
+import com.datadog.android.core.internal.utils.toJsonArray
+import com.example.forgery.ForgeryConfiguration
+import com.google.gson.JsonArray
+import com.google.gson.JsonElement
+import com.google.gson.JsonNull
+import com.google.gson.JsonObject
+import fr.xgouchet.elmyr.junit4.ForgeRule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.util.Date
+
+class ModelWithAdditionalAttributesTest {
+
+    @get:Rule
+    val forge = ForgeRule()
+
+    @Before
+    fun `set up`() {
+        ForgeryConfiguration().configure(forge)
+    }
+
+    @Test
+    fun `validate model for additionalProperties with Any? key value type`() {
+        val type = Class.forName("com.example.model.Company")
+        val toJson = type.getMethod("toJson")
+        val fromJson = type.getMethod("fromJson", String::class.java)
+        repeat(10) {
```

Comment:
why `repeat` is needed?

Reply:
but why it is needs to be executed several times? some part of it is not deterministic?